packages/frontend/src/js/app/pages/Account/index.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <PrivateLayout class="mb-16">
+        <LoadingError v-if="error || (!user && !loading)">
+            L'utilisateur n'existe pas en base de données.
+        </LoadingError>
+        <div v-else>
+            <AccountRead v-if="!edit" :user="user" @openEdit="edit = true" />
+            <AccountEdit v-else :user="user" @cancelEdit="edit = false" />
+        </div>
+    </PrivateLayout>
+</template>
+
+<script>
+import PrivateLayout from "#app/components/PrivateLayout";
+import LoadingError from "#app/components/PrivateLayout/LoadingError.vue";
+import AccountRead from "./AccountRead/AccountRead";
+import AccountEdit from "./AccountEdit/AccountEdit";
+import { get as getConfig } from "#helpers/api/config";
+import { get as getUser } from "#helpers/api/user";
+
+export default {
+    components: {
+        PrivateLayout,
+        LoadingError,
+        AccountRead,
+        AccountEdit
+    },
+    data() {
+        return {
+            user: null,
+            edit: false,
+            loading: false,
+            error: null
+        };
+    },
+    methods: {
+        async load() {
+            this.loading = true;
```

Comment:
Il faudrait également passer `this.edit` à `false`, et `this.error` à `null` non ?